content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md

```diff
@@ -30,25 +52,23 @@ For basic operations like accessing an instance for these steps, see [managed in
 1. In `deploy-sourcegraph-managed/$COMPANY` run `terraform init && terraform apply`
-1. Access the instance over SSH and confirm all containers are healthy ([instructions](operations.md#ssh-access)). You may find `docker ps` reports no containers, that indicates it is still installing Docker, etc. To watch this progress see [debugging startup scripts](operations.md#debugging-startup-scripts), it usually takes <10m. Confirm all containers come up healthy (`docker ps` should report them as such)
+1. Check if instance is ready
+
+```
+mg check
+```
+
 1. In the infrastructure repository, [create a DNS entry](https://github.com/sourcegraph/infrastructure/blob/main/dns/sourcegraph.managed.tf) that points `$COMPANY.sourcegraph.com` to the `default-global-address` IP (see ["Finding the external load balancer IP"](operations.md#finding-the-external-ips)) and follow the process there to `asdf exec terraform apply` it. If the instance is Public, set `proxied` to `true`. If it's Private, set it to `false`.
-1. Create a PR for review.
-1. Create admin credentials in 1password:
-   - Open the 1password [Managed instances vault](https://my.1password.com/vaults/l35e5xtcfsk5suuj4vfj76hqpy/allitems) (if necessary, access can be requested in #it-tech-ops)
-   - **Add** > **Login** > enter **$COMPANY sourcegraph-admin** as the title
-     - **User:** `managed+$COMPANY@sourcegraph.com`
-     - **Password:** Change **length** to 40 and turn on symbols and digits > **Save**
-1. Access the Sourcegraph web UI ([instructions for port-forwarding](operations.md#port-forwarding-direct-access-to-caddy-jaeger-and-grafana))
-1. Set up the initial admin account (for use by the Sourcegraph team only)
-   - Email: `managed+$COMPANY@sourcegraph.com` (note `+` sign not `-`)
-   - Username: `sourcegraph-admin`
-   - Password: Use the password previously created and stored in 1password.
-1. Create a token for the account under `/users/sourcegraph-admin/settings/tokens` called `managed-instances` with `user:all` (i.e. the default) checked and add it as "token" under the 1password entry for the admin account.
+1. In the GCP web UI under **Network services** > **Load balancers** > select the load balancer > watch the SSL certificate status. It may take some time for it to become active (~1h41m) / for Google to see the DNS change from Cloudflare. Confirm it is active by following ["Access through the GCP load balancer as a user would"](operations.md#access-through-the-gcp-load-balancer-as-a-user-would).
+1. Create a PR for review, apply and merge
+1. Initialise instance
+
+```
+mg init-instance $CUSTOMER_ADMIN_EMAIL
+```
+
 1. Navigate to Grafana under `/-/debug/grafana` and confirm the instance looks healthy.
-1. Configure `externalURL` in the site configuration, and use SSH to restart the server (`sudo su`, `shutdown -r`) wait for it to come back up and access it again.
 1. In the **global user settings** (not site config), set `"alerts.showPatchUpdates": false`
```

Comment:
Should we put 3 next steps also in `mg init-instance` command? this would simplify the flow, and they are always the same.

Reply:
I will make separate PR for it - same as for @michaellzc `mg create-oidc-secrets` proposal not to make too big PR